modules/ROOT/pages/dw-values-functions-update.adoc

```diff
@@ -286,54 +286,6 @@ output application/json
  }
 ----
 
-=== Example
-
-This example updates one of the recurring fields.
-
-==== Input
-
-[source,XML,linenums]
-----
-<users>
-  <user>
-    <name>Phoebe</name>
-    <language>French</language>
-  </user>
-  <user>
-    <name>Joey</name>
-    <language>English</language>
-  </user>
-</users>
-----
-
-==== Source
-
-[source,DataWeave,linenums]
-----
-%dw 2.0
-import * from dw::util::Values
-output application/xml
----
-payload update ["users", "user", "language"] with (if ($ == "English") "Gibberish" else $)
-----
-
-==== Output
-
-[source,XML,linenums]
-----
-<users>
-  <user>
-    <name>Phoebe</name>
-    <language>French</language>
-  </user>
-  <user>
-    <name>Joey</name>
-    <language>Gibberish</language>
-  </user>
-</users>
-----
-
-
```

Comment:
Do you think this example needs to be added to the data-weave repo? Or was it deleted intentionally?

Reply:
I didn't find references in the data-weave repo about this example, maybe it was added in docs repository only. You can add it to the data-weave repo.